src/Skia/Perspex.Skia/SkiaPlatform.cs

```diff
@@ -30,7 +30,10 @@ public static bool ForceSoftwareRendering
             set
             {
                 s_forceSoftwareRendering = value;
-                MethodTable.Instance.SetOption(MethodTable.Option.ForceSoftware, new IntPtr(value ? 1 : 0));
+
+                // Do we still need this with SkiaSharp??
```

Comment:
I get yah, it's just hard to wrap all of this change up in a single perfect PR that leaves in a happy state. This port as is most things in Perspex is a work in progress. :)